### PR TITLE
i18n(es): Updated `layouts.mdx`,`cloudflare.mdx` & `react.mdx`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- What does this PR change? Give us a brief description.
Updated `layouts.mdx`,`cloudflare.mdx` & `react.mdx` based on these commits:
`layouts.mdx` -> bd877303a6ce3ef1ace9c85688df15cd1ec0a1f6 & e832eedb3c34c29630e753b6e46135d366fcf7f0
`cloudflare.mdx` -> b2fb8f8afabab3d113658e7e659cf3ca2e2a73dd
`react.mdx` -> 4b33a6231bb5cf245fc6904a1cb011c6a8ddc682

### CODE REVIEW

File: src/content/docs/es/guides/integrations-guide/react.mdx [hunk 1]
```diff
@@ -73,6 +73,45 @@ Para usar tu primer componente React en Astro, dirígete a nuestra [documentaci
 *   💧 opciones de hidratación del lado del cliente, y
 *   🤝 oportunidades para mezclar y anidar frameworks juntos
 
+## Opciones
+
+### Anàlisis de los hijos
+
+Los hijos pasados a un componente React desde un componente Astro se analizan como cadenas simples, no como nodos de React.
```
Comment (Waxer59) [inline src/content/docs/es/guides/integrations-guide/react.mdx:80]:
```suggestion
Los hijos pasados a un componente de React desde un componente de Astro se analizan como strings simples, no como nodos de React.
```

**Review by Waxer59 — COMMENTED** [review]
Buena PR 🤠, te dejo unas sugerencias

**Comment by dreyfus92** [comment]
Muchas gracias por la revisión @Waxer59 😁

**Review by Waxer59 — APPROVED** [review]
LGTM ✔

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks peeps 🚀

**Pull request merged**